resources/ddl/define-all.hive

```diff
@@ -507,6 +507,9 @@ create temporary function map_get as 'hivemall.tools.map.MapGetUDF';
 drop temporary function if exists map_key_values;
 create temporary function map_key_values as 'hivemall.tools.map.MapKeyValuesUDF';
 
+drop temporary function if exists map_roulette;
```

Comment:
Please use `bin/update_ddls.sh` to update all relevant DDLs.

See https://github.com/apache/incubator-hivemall#contributing